privacyidea/lib/challengeresponsedecorators.py

```diff
@@ -59,9 +59,14 @@ def _create_challenge(token_obj, challenge_type, message, challenge_data=None):
     db_challenge.save()
     token_obj.challenge_janitor()
     reply_dict = {}
+    if ((challenge_type == CHALLENGE_TYPE.PIN_RESET) or (challenge_type == CHALLENGE_TYPE.RESYNC)):
+        challenge = challenge_type
+    else:
+        challenge = "standard"
```

Comment:
This should be added to the [`CHALLENGE_TYPE` class](https://github.com/privacyidea/privacyidea/blob/28f138522618915950769a0fe196720f3b417393/privacyidea/lib/challengeresponsedecorators.py#L46), maybe as `generic_challenge` or `standard_challenge` (@cornelinux should know, why the values are called `generic_*`).

Reply:
@lukasmatusiewicz is 'generic_challenge' instead of 'standard' fine for you?

Reply:
7d835b180677b94d0de6d320c9d8841caff697a1